internal/cmd/connect/command_plugin_install.go

```diff
@@ -205,20 +200,20 @@ func parsePluginId(plugin string) (string, string, string, error) {
 func getLocalManifest(archivePath string) (*manifest, error) {
 	zipReader, err := zip.OpenReader(archivePath)
 	if err != nil {
-		return nil, errors.Errorf("failed to open local archive file %s: %v", archivePath, err)
+		return nil, errors.Wrapf(err, "failed to open local archive file %s", archivePath)
 	}
 	defer zipReader.Close()
 
 	for _, zipFile := range zipReader.File {
 		isManifest, err := filepath.Match("*/manifest.json", filepath.ToSlash(zipFile.Name))
 		if err != nil {
-			return nil, errors.Errorf("failed to examine file %s inside local archive file %s: %v", zipFile.Name, archivePath, err)
+			return nil, errors.Wrapf(err, "failed to examine file %s inside local archive file %s", zipFile.Name, archivePath)
 		}
 
 		if isManifest {
 			manifestFile, err := zipFile.Open()
 			if err != nil {
-				return nil, errors.Errorf("failed to open manifest file %s inside local archive file %s: %v", zipFile.Name, archivePath, err)
+				return nil, errors.Wrapf(err, "failed to open manifest file %s inside local archive file %s", zipFile.Name, archivePath)
```

Comment:
We may want to do this across the whole CLI?

Reply:
Should be a quick followup. Took a quick look; there's only a handful of spots where `Errorf` is used to format an existing error like this.

Reply:
Don't forget `fmt.Errorf()`!